packages/backend/src/api/request.ts

```diff
@@ -180,6 +183,11 @@ function getTraceId(data: unknown, headers?: Headers): string {
   return cfRay || '';
 }
 
+function getRetryAfter(headers?: Headers): number | undefined {
+  const retryAfter = headers?.get('Retry-After');
+  return retryAfter ? parseInt(retryAfter, 10) : undefined;
```

Comment:
Should we also handle NaN? 

```suggestion
  return retryAfter ? (isNaN(parseInt(retryAfter, 10)) ? undefined : parseInt(retryAfter, 10)) : undefined;
```

Reply:
good catch!